app/src/main/java/com/zeapo/pwdstore/UserPreference.kt

```diff
@@ -374,6 +376,22 @@ class UserPreference : AppCompatActivity() {
             }
         }
 
+        private fun updateClearSavedPassphrasePrefs() {
+            clearSavedPassPreference?.apply {
+                val sshPass = encryptedPreferences.getString("ssh_key_local_passphrase", null)
+                val httpsPass = encryptedPreferences.getString("https_password", null)
+                if (sshPass == null && httpsPass == null) {
+                    isVisible = false
+                    return
+                }
+                title = when {
+                    sshPass != null -> getString(R.string.clear_saved_passphrase_ssh)
+                    httpsPass != null -> getString(R.string.clear_saved_passphrase_https)
+                    else -> null
+                }
+            }
```

Comment:
I think that this needs to call some version of [`notifyChanged()`](https://developer.android.com/reference/androidx/preference/Preference#notifyChanged())

Reply:
It's not available as public API in Preference objects.

Reply:
True... Okay, I don't know why you are experiencing this issue.